packages/quantic/cypress/e2e/default-1/case-classification/case-classification.cypress.ts

```diff
@@ -889,4 +893,343 @@ describe('quantic-case-classification', () => {
       });
     });
   });
+
+  describe('when a field value changes and the fetchClassificationsOnChange property is set to true', async () => {
```

Comment:
This describe text needs to be updated no?

Wouldn't it be:
`when the fetchClassificationsOnChange property is set to true`